apps/faucet/.env.sample

```diff
@@ -1,9 +1,8 @@
-# Faucet API Endpoint override
+# Faucet API URL Default
 NAMADA_INTERFACE_FAUCET_API_URL=http://127.0.0.1:5000
-NAMADA_INTERFACE_FAUCET_API_ENDPOINT=/api/v1/faucet
 
-# Faucet limit, as defined in genesis toml
-NAMADA_INTERFACE_FAUCET_LIMIT=1000
```

Comment:
can you remove this env var also from `apps/faucet/README.md` and `release-faucet.yml`

Reply:
Thanks for catching that!